### DISCUSSION TITLE
How to solve osd isn't created?

### DISCUSSION THREAD

**Discussion by integrous** [rook/rook#11553] [Q&A]
this is osd-prepare log error:

cephosd: failed to determine if the encrypted block "" is from our cluster. failed to dump LUKS header for disk "". Device does not exist or access denied.: exit status 4

**Comment by travisn** [comment]
That message is probably not the cause. See [this topic](https://rook.io/docs/rook/latest/Troubleshooting/ceph-common-issues/#osd-pods-are-not-created-on-my-devices) for more ideas to investigate the OSD prepare logs for why the OSDs are not created.
